cfgov/unprocessed/css/video-player.less

```diff
@@ -162,59 +162,68 @@
         }
     }
 
-    .share-icon {
+    .m-social-media_icon {
         color: @pacific;
     }
 
-    .respond-to-min(@bp-sm-min, {
-        text-align: right;
-    });
+    .m-social-media_icons {
+    	display: block;
+    	margin-right: unit( 5px / @base-font-size-px, em);
+    }
 }
 
-.video-player_share-actions,
+.video-player .m-social-media,
 .video-player_video-actions {
-    .respond-to-max(@bp-xs-max, {
+    .respond-to-max( @bp-xs-max, {
         margin-top: @gutter-half__em;
-    });
+    } );
 
-    .respond-to-range(@bp-sm-min, @fcm-bp-max {
+    .respond-to-range( @bp-sm-min, @fcm-bp-max {
         .grid_column(6);
-    });
+    } );
 }
 
-.respond-to-max(@fcm-bp-max, {
+.respond-to-max( @fcm-bp-max, {
     .video-player_iframe-container {
         .u-flexible-container();
     }
 
     .video-player_iframe {
         .u-flexible-container_inner();
     }
-});
+} );
 
-.respond-to-min(@fcm-bp-min, {
+.respond-to-min( @fcm-bp-min, {
     .video-player_video-actions-container {
         position: absolute;
         top: 0;
         left: @video-width__px;
     }
 
-    .video-player_share-actions,
+    .video-player .m-social-media,
     .video-player_video-actions {
         margin-bottom: @grid_gutter-width;
     }
-    .video-player_share-actions{
+
+    .video-player .m-social-media {
         text-align: left;
     }
-});
+} );
 
-.respond-to-min(@bp-xl-min, {
+.respond-to-min( @bp-xl-min, {
     .video-player_iframe-container {
         width: @video-width__px;
         margin: 0 auto;
     }
+
     .video-player_video-actions-container {
         right: 0;
         left: auto;
     }
-});
+} );
+
+.respond-to-range( @bp-xs-max, @fcm-bp-min, {
+	.video-player .m-social-media {
+		text-align: right;
+	}
+} );
```

Comment:
Trivial: Please use 4-space indents here. This block's using tabs, which are rendering in the classic **8**-character width of the GitHub diff view.